tests/test_queries.py

```diff
@@ -263,7 +264,128 @@ def test_bus(client: FlaskClient) -> None:
+        "ArrivalTime",
+    )
+    assert json["answer"]
+    assert all(not c.isdecimal() for c in json["voice"])
+
+    json = qmcall(
+        client,
+        {
+            "q": "hvaða leiðir stoppa í Bíó Paradís?",
+            "voice": True,
+            "private": False,
+        },
+        "WhichRoute",
+    )
+    assert (
+        json["answer"]
+        and "stoppar" not in json["answer"]
+        and "stoppa" in json["answer"]
+        #   (^ Multiple routes go through Bíó Paradís)
+    )
+    assert all(not c.isdecimal() for c in json["voice"])
+    # Following query relies on the query above
+    json = qmcall(
+        client,
+        {"q": "hvenær kemur strætisvagn fjórtán þangað?", "voice": True},
+        "ArrivalTime",
+    )
+    assert json["answer"] and "Bíó Paradís" in json["answer"]
+    assert (
+        all(not c.isdecimal() for c in json["voice"])
+        and "Bíó Paradís" in json["voice"]
+        and "fjórtán" in json["voice"]
+    )
+
+    json = qmcall(
+        client,
+        {"q": "Klukkan hvað stöðvar sexan á Hlemmi?", "voice": True},
+        "ArrivalTime",
+    )
+    assert json["answer"]
+    assert all(not c.isdecimal() for c in json["voice"])
+
+    json = qmcall(
+        client,
+        {
+            "q": "hvaða leiðir stoppa á Mýrarvegi Hringteig?",
+            "voice": True,
+        },
+        "WhichRoute",
+    )
+    assert "Mýrarvegi / Hringteig" in json["answer"]
+    assert all(not c.isdecimal() for c in json["voice"])
+
+    import straeto
+    if len(straeto.BusStop.named("Naustabraut / Davíðshagi A")) == 0:
+        # Stop doesn't exist in old stops.txt file in straeto package
+        return
+
+    json = qmcall(
+        client,
+        {
+            "q": "hvaða leiðir stoppa á Naustabraut Davíðshagi A?",
+            "voice": True,
+        },
+        "WhichRoute",
+    )
+    assert "Naustabraut / Davíðshaga A" in json["answer"]
+    assert (
+        all(not c.isdecimal() for c in json["voice"])
+        and "Naustabraut / Davíðshaga austur" in json["voice"]
+    )
+
+    json = qmcall(
+        client,
+        {
+            "q": "hvaða leiðir stoppa á Naustabraut Davíðshaga A?",
+            "voice": True,
+        },
+        "WhichRoute",
+    )
+    assert "Naustabraut / Davíðshaga A" in json["answer"]
+    assert (
+        all(not c.isdecimal() for c in json["voice"])
+        and "Naustabraut / Davíðshaga austur" in json["voice"]
+    )
+
+    # json = qmcall(
+    #     client,
+    #     {
+    #         "q": "hvaða leiðir stoppa á Naustabraut Davíðshaga austur?",
+    #         "voice": True,
+    #     },
+    #     "WhichRoute",
+    # )
+    # assert "Naustabraut / Davíðshaga A" in json["answer"]
+    # assert (
+    #     all(not c.isdecimal() for c in json["voice"])
+    #     and "Naustabraut / Davíðshaga austur" in json["voice"]
+    # )
+
+    _query_data_cleanup()  # Remove any data logged to DB on account of tests
```

Comment:
I thought we had somewhere a test parameter that could be set on the query module which prevents logging?

Reply:
In my experience it can be a bit flaky sometimes. When I interrupt a running test, sometimes stuff ends up in the database. This at least explicitly cleans up db stuff when the bus test function finishes.
I will look into this issue soon though (and verify whether or not I'm just imagining things being added to the db :) ).